### ISSUE TITLE
Accessibility swarm on the Script Link component

### ISSUE THREAD

**Issue by DenisHdz** [bbc/simorgh#4180]
**Is your feature request related to a problem? Please describe.**
 Carry out a `two devs` accessibility swarm for the `ScriptLink` component.

**Describe the solution you'd like**
Go through this [checklist](https://bbc.github.io/accessibility-news-and-you/accessibility-news-and-testers) and fix any issue we might face.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by amywalkerdev** [comment]
Needs #2454 to be done then we can do an a11y swarm

**Comment by ghost** [comment]
The `ScriptLink` overlaps `Brand`. Breakpoints  < 332

<img width="287" alt="Screenshot 2019-10-31 at 13 00 56" src="https://user-images.githubusercontent.com/48688870/67937683-0f086d00-fbdf-11e9-91e2-859da2466c87.png">

**Comment by Bopchy** [comment]
Currently, `when navigating through the banner landmark on a front page with a script link, the script link is announced before the ‘skip to content’ link`. This will be fixed in https://github.com/bbc/psammead/issues/2713 , such that the script link is announced `after` the ‘skip to content’ link

**Comment by DenisHdz** [comment]
To be consistent with the Nav menu button styling, UX has requested to update the `ScriptLink` hover/focus state https://github.com/bbc/psammead/pull/2685.

**Comment by DenisHdz** [comment]
The `Skip to content` link is not well located in IE.

![skip-to-content](https://user-images.githubusercontent.com/46446236/69793396-3cd1d900-11c0-11ea-84a9-de3c844bea16.jpg)

This has been fixed in https://github.com/bbc/psammead/pull/2680

**Comment by Bopchy** [comment]
https://github.com/bbc/simorgh/issues/4180#issuecomment-548299007 was fixed in https://github.com/bbc/psammead/issues/2534

**Comment by Bopchy** [comment]
https://github.com/bbc/psammead/issues/2751 was discovered during the swarm. This will be fixed in https://github.com/bbc/psammead/pull/2750

https://github.com/bbc/psammead/pull/2750 also adjusts the `height`, `line-height` and `padding` of the ScriptLink for `@media (min-width: 400px)`

**Comment by tochwill** [comment]
I've got the devices in the office, will finish these checks off now

**Comment by tochwill** [comment]
I have completed the rest of the assistive technology checks - text is announced as 'Lat' rather than 'Latin' as in the acceptance criteria (as the text written is 'Lat'), so will check if this is OK. I am unable to get Read&Write to speak the text on either IE11 or Chrome, so will look further into this as well.

**Comment by ghost** [comment]
I guess it is because it is not wrapped in a container with `aria-hidden` attribute. In the storybook we could have 
```jsx
<ScritpLink>
  <span aria-hidden='true'> Lat </span>
   <VisuallyHiddenText> Latin </VisuallyHiddenText>
</ScriptLink>
```

**Comment by tochwill** [comment]
> I guess it is because it is not wrapped in a container with `aria-hidden` attribute. In the storybook we could have
> 
> ```js-jsx
> <ScritpLink>
>   <span aria-hidden='true'> Lat </span>
>    <VisuallyHiddenText> Latin </VisuallyHiddenText>
> </ScriptLink>
> ```

This works :)

**Comment by tochwill** [comment]
Read&Write is not reading the script link on IE11 on windows. On Chrome, on hovering over the script link, all of the content of the Brand component is read. A javascript error is then thrown by the browser extension, and the link is not read again on a second hover.

<img width="423" alt="Screenshot 2019-12-04 at 17 47 33" src="https://user-images.githubusercontent.com/6690955/70167254-3959b880-16be-11ea-952d-1f2822c51680.png">

**Comment by ghost** [comment]
Read & write issue fixed in https://github.com/bbc/psammead/pull/2833

**Comment by DenisHdz** [comment]
This can be closed since all the issues/bugs have been fixed.
